src/modules/scene/utils.ts

```diff
@@ -40,3 +40,18 @@ export function snapToGrid(position: Vector3, grid: number = 0.5): Vector3 {
     z: position.z - (position.z % grid)
   }
 }
+
+export function clearGround(groundId: string, entities: Scene['entities']): Scene['entities'] {
```

Comment:
I would rename this to something like `getEntitiesWithoutGrounds` or something better, clearGround just sounds mutate-y